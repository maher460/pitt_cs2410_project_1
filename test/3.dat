ori R1, R0, -20
L1: addi R1, R1, 1
bne R1, $0, L1