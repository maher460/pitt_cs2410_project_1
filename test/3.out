Basic Branch

----------
Data Cache
----------
-----------------
Integer Registers
-----------------
Int_Reg[0]=0
Int_Reg[1]=0
Int_Reg[2]=0
Int_Reg[3]=0
Int_Reg[4]=0
Int_Reg[5]=0
Int_Reg[6]=0
Int_Reg[7]=0
Int_Reg[8]=0
Int_Reg[9]=0
Int_Reg[10]=0
Int_Reg[11]=0
Int_Reg[12]=0
Int_Reg[13]=0
Int_Reg[14]=0
Int_Reg[15]=0
Int_Reg[16]=0
Int_Reg[17]=0
Int_Reg[18]=0
Int_Reg[19]=0
Int_Reg[20]=0
Int_Reg[21]=0
Int_Reg[22]=0
Int_Reg[23]=0
Int_Reg[24]=0
Int_Reg[25]=0
Int_Reg[26]=0
Int_Reg[27]=0
Int_Reg[28]=0
Int_Reg[29]=0
Int_Reg[30]=0
Int_Reg[31]=0
------------------------
Floating Point Registers
------------------------
FP_Reg[0]=0.0
FP_Reg[1]=0.0
FP_Reg[2]=0.0
FP_Reg[3]=0.0
FP_Reg[4]=0.0
FP_Reg[5]=0.0
FP_Reg[6]=0.0
FP_Reg[7]=0.0
FP_Reg[8]=0.0
FP_Reg[9]=0.0
FP_Reg[10]=0.0
FP_Reg[11]=0.0
FP_Reg[12]=0.0
FP_Reg[13]=0.0
FP_Reg[14]=0.0
FP_Reg[15]=0.0
FP_Reg[16]=0.0
FP_Reg[17]=0.0
FP_Reg[18]=0.0
FP_Reg[19]=0.0
FP_Reg[20]=0.0
FP_Reg[21]=0.0
FP_Reg[22]=0.0
FP_Reg[23]=0.0
FP_Reg[24]=0.0
FP_Reg[25]=0.0
FP_Reg[26]=0.0
FP_Reg[27]=0.0
FP_Reg[28]=0.0
FP_Reg[29]=0.0
FP_Reg[30]=0.0
FP_Reg[31]=0.0

commit instructions:41